ui/.neutrinorc.js

```diff
@@ -157,12 +159,42 @@ module.exports = {
           .test(/taskcluster-version$/)
           .use('raw-loader')
             .loader('raw-loader');
-
     },
     (neutrino) => {
       neutrino.config.resolve
         .alias.set('taskcluster-ui', resolve(__dirname, 'src/'));
     },
+    (neutrino) => {
+      neutrino.use(copy, {
+        patterns: [
+          {
+            context: 'src/static',
+            from: '**/*',
+            to: 'static',
+          },
+        ],
+      });
+
+      // Generate env.js if it doesn't exist
+      if (process.env.GENERATE_ENV_JS) {
+        const envJs = `window.env = ${
+          JSON.stringify(
+            Object.keys(ENV_DEFAULTS).reduce((acc, curr) => {
+              acc[curr] = process.env[curr] || ENV_DEFAULTS[curr];
+
+              return acc;
+            }, {}), null, 2)
+        };`;
+
+        if (!fs.existsSync(STATIC_DIR)){
+          fs.mkdirSync(STATIC_DIR);
+        }
+
+        if (!fs.existsSync(join(STATIC_DIR, 'env.js'))){
+          fs.writeFileSync(join(STATIC_DIR, 'env.js'), envJs, 'utf8');
+        }
+      }
```

Comment:
let's unlink it here, just so that we never end up accidentally including something in a production build.

Reply:
I'm not sure I understand the issue. Are you suggesting deleting `static/env.js` after L196? In what scenario would we be accidentally including something else?